sentry_sdk/transport.py

```diff
@@ -448,6 +464,8 @@ def _get_pool_options(self, ca_certs):
 
         if self.options["socket_options"]:
             options["socket_options"] = self.options["socket_options"]
+        elif self.options["keep_alive"]:
+            options["socket_options"] = KEEP_ALIVE_SOCKET_OPTIONS
```

Comment:
Wouldnt it be nice that people can set `keep_alive=True` but then also override some values in `socket_options`?

Reply:
Added here: https://github.com/getsentry/sentry-python/pull/2842/commits/19534decf4512c7db132d6fd07c42c408a847930